src/jwplayer.jsx

```diff
@@ -43,23 +70,35 @@ class JWPlayer extends React.Component {
     return this.playerLoadPromise.then(() => window.jwplayer(view.id).setup(setupConfig));
   }
 
+  didOnEventsChange(nextProps) {
+    const onEventFilter = (prop) => prop.match(ON_REGEX);
+    const currEvents = Object.keys(this.props).filter(onEventFilter);
+    const nextEvents = Object.keys(nextProps).filter(onEventFilter);
+    const newEvents = nextEvents.some((event) => currEvents.indexOf(event) === -1
```

Comment:
We can sort both of these arrays when they get assigned above. Then we can check if each index matches, rather than using `indexOf`. This will net us better performance as we won't have a loop within a loop.
```suggestion
    const newEvents = nextEvents.some((event, index) => currEvents[i] !== event
```